### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/virtual-environments.md`

### PR DESCRIPTION
**Pull request by marcelomarkus**
- Portuguese translation
docs/pt/docs/virtual-environments.md

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]

File: docs/pt/docs/virtual-environments.md [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+---> 100%
+```
+
+</div>
+
+////
+
+//// tab | `uv`
+
+Se você tem <a href="https://github.com/astral-sh/uv" class="external-link" target="_blank">`uv`</a>:
```
Comment (ceb10n) [inline docs/pt/docs/virtual-environments.md:323]: I would add *o* here:

*"Se você tem **o** `uv`"*

* just a suggestion 😄

File: docs/pt/docs/virtual-environments.md [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
```
Comment (ceb10n) [inline docs/pt/docs/virtual-environments.md:355]: the same here:

*"Se você tem **o** `uv`"*

**Review by ceb10n — APPROVED** [review]
Great work 🚀 

* just added some suggestions, but they are not impediments to approval

**Review by nazarepiedady — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for your work! @marcelomarkus :medal_sports: 
 
And thanks to all for the reviews :mag:

**Pull request merged**